### DISCUSSION TITLE
External editor usage flow

### DISCUSSION THREAD

**Discussion by gazmanic** [DaxStudio/DaxStudio#783] [Ideas]
Using external editor (vim) along with the internal VAX Studio (VS) one.  Would be nice if VS recognised when a file is dirty and/or had provision for external editors.  Any changes externally to a file needs the File Close > File Open > Reconnect data source rather than just reload/refresh the file into VS.

**Comment by dgosbell** [comment]
So this is a forum for DAX Studio, not VAX Studio. Is this just a typo or are you looking for a different product?

If you are talking about DAX Studio I'd be curious to know why you would need to use an external editor in the first place. This is not something I've ever felt the need for myself or something that has been asked for before. So I'm curious try and figure out how widely used this sort of usage flow is likely to be and how many others it might benefit.

**Reply by gazmanic** [reply]
Hilarious, yes DAX Studio ... vim on the brain :)   What code editor do you use?  Have you used vim or emacs?  Are you happy to reduce mouse usage?  Thank you for the quick response :)

**Comment by dgosbell** [comment]
When I'm writing DAX I use DAX Studio, I don't really use the mouse much already.

**Reply by gazmanic** [reply]
All good, sounds like not much need for how I am using the tool.  For minor changes obviously go straight to the DAX Studio editor but for more than minor edits it is easier to use a more powerful code editor.
Merry Christmas bro :)
